Assets/PatchKit Patcher/Scripts/Patcher.cs

```diff
@@ -899,6 +899,10 @@ private void ThreadUpdateApp(bool automatically, CancellationToken cancellationT
             {
                 _appInfo.Value = _app.RemoteMetaData.GetAppInfo(!automatically, _updateAppCancellationTokenSource.Token);
                 _remoteVersionId.Value = _app.GetLatestVersionId(!automatically, _updateAppCancellationTokenSource.Token);
+                var latestAppVersion =
+                    _app.RemoteMetaData.GetAppVersionInfo(_remoteVersionId.Value.Value, false, cancellationToken);
+                _app.LocalMetaData.SetMainExecutableAndArgs(latestAppVersion.MainExecutable, latestAppVersion.MainExecutableArgs);
```

Comment:
I suggest doing it after a successful Update operation instead of at the start.

Reply:
done